src/main/metanorma/sections/informative/generic_name.adoc

```diff
@@ -27,7 +27,7 @@ image::path_components.svg["urn:ogc:def:crs:EPSG:10" components]
 
```

Comment:
@desruisseaux there are a couple of instances of image:: [that don't appear](https://docs.ogc.org/DRAFTS/23-016.html#generic_name) in the html output, like the one on line 26 above.  I think it's because of the quoted string inside the brackets i.e. `["urn:ogc:def:crs:EPSG:10" components]`

Reply:
Thanks for spotting that! I just tried a fix. We will need to wait for the OGC server to regenerate the document to see if it works.